### PR TITLE
fixes the save button to the side

### PR DESCRIPTION
**Pull request by Clayseph**

### CODE REVIEW

File: src/containers/CharacterSheet.js [hunk 1]
```diff
@@ -78,87 +78,91 @@ const CharacterSheet = ({ characterData }) => {
...
+          ) }
         </TopAnchor>
       ) }
       { editMode && (
         <Admin>
-          <Row style={ { justifyContent: 'flex-end' } }>
-            <Button onClick={ cancel }>Cancel</Button>
-            <Button onClick={ save } variant="contained" color="secondary">
+          <Row style={{
+            position: 'fixed', justifyContent: 'flex-end', width: '65%',
```
Comment (Clayseph) [inline src/containers/CharacterSheet.js]: only actual change

**Comment by kjintroverted** [comment]
That one looks good, but thats only when they make edits to the core character. do we want to pin the save button for all other edits as well?
Screen Shot 2019-12-20 at 3 27 29 PM

**Comment by kjintroverted** [comment]
Also, in edit, the save is pinned but is hidden under the other components
Screen Shot 2019-12-20 at 3 30 01 PM

**Comment by Clayseph** [comment]
What's your screensize? haha mine is all the way to the side. I'll slap a z index in there

Also totally forgot about the other one

**Comment by kjintroverted** [comment]
Looking better, but mobile is rough. I think these may need a background and maybe some padding at the top to make sure they're not on top of each other all the time
image

**Pull request merged**